k2/csrc/ragged_ops.h

```diff
@@ -1346,6 +1346,9 @@ Array1<T> ComputeHash(Ragged<int32_t> &src);
                        repeats (i.e., multiplicity) of each output sequence.
                        The caller does not need to pre-allocate it. It is
                        allocated inside the function.
+     @param [out] new2old_indexes
```

Comment:
Each output sublist could have appeared multiple times in the input, so is there a rule for which one is listed?

Should probably specify that this would be an idx0 if `src` had 2 axes, and an idx01 if `src` had 3 axes.

Reply:
> Each output sublist could have appeared multiple times in the input, so is there a rule for which one is listed?

Sublists of a seq are reordered by their hash values by calling `SortSublists`.
Although the implementation of`SortSublists` does not mention that it is stable, the Python tests
show that the first one is kept, other repeats are discarded.

I think we don't care which one is kept since their contents are identical (assuming there are no hash conflicts).

Reply:
> Should probably specify that this would be an idx0 if src had 2 axes, and an idx01 if src had 3 axes.

Thanks. Done.

Reply:
If it just puts an arbitrary one of the inputs, you should specify that.  (Also specify if this is nondeterministic, i.e.
may give different outputs for the same input on different run.s).

Reply:
> If it just puts an arbitrary one of the inputs, you should specify that. (Also specify if this is nondeterministic, i.e.
may give different outputs for the same input on different run.s).

Fixed. It is described in the comment that it uses a deterministic algorithm.